debug-tools/gce-serial-output-continuously-append.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+
+INSTANCE="$1"
+
+if [ -z "$INSTANCE" ]; then
+    echo "first argument must be the GCE instance (for example, mar280632-365378)"
+    exit 1
+fi
+
+# wait for the instance to exist
+until gcloud compute --project=snapd-spread instances describe "$INSTANCE" --zone=us-east1-b >/dev/null 2>&1; do 
+    echo "waiting for instance to exist"
+    sleep 1
+done
+
+backgroundscript=/tmp/background-$RANDOM.sh
```

Comment:
(nitpick) this could be: ` backgroundscript=$(mktemp --suffix=.gce-watcher)`

Reply:
Done!